### PR TITLE
Fix for building the wheel file - changed name to primeqa, added authors list, url, license.

### PR DESCRIPTION
**Pull request by hansolosan**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,14 +23,28 @@
...
+    "Saneem A Chemmengath <saneem.cg@in.ibm.com>",
+    "Anthony Ferritto",
 ]
 
 _deps = {
     "docutils>=0.14,<0.18": ["tests"],
     "bitarray~=2.3.7": ["install"],
     "bump2version~=1.0.1": ["dev"],
     "click~=8.0.4": ["install"],
+    "cupy-cuda113~=10.4.0": ["install"],
```
Comment (santhnm2) [inline setup.py:47]: Was this necessary for something other than ColBERT?

Comment (avisil) [reply setup.py:47]: I don't think so. @franzm @bhavani105 @jdpsen @jsmcibm ?

Comment (bhavani105) [reply setup.py:47]: The setup.py needs to be synched with the version in main.

Comment (jdpsen) [reply setup.py:47]: not in QG or QA.

**Review by jdpsen — APPROVED** [review]
LGTM

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,20 @@
 ]
 
 authors = [
-    "TODO"
+    "Bhavani Iyer <bsiyer@us.ibm.com>",
+    "Avirup Sil <avi@us.ibm.com>",
+    "Martin Franz <franzm@us.ibm.com>",
+    "Mihaela Bornea <mabornea@us.ibm.com>",
+    "Sara Rosenthal <sjrosenthal@us.ibm.com>",
+    "Avirup Sil <avi@us.ibm.com>",
```
Comment (avisil) [inline setup.py:32]: somehow my name is twice is here? We should remove.

**Review by avisil — APPROVED** [review]
This can be merged. I left a minor comment which we can address in the future.

**Pull request merged**